rfcs/rfc-standalone-annual-report.md

```diff
@@ -0,0 +1,77 @@
+
+
+# Summary
+
+Proposal:  create a standalone Annual Report filing that will contain the first instance of  the proposed "Register a beneficial owner" workflow. 
+
+# Motivation
+
+1. Creating a standalone AR filing that includes beneficial owner registration delivers on one of the key selling points made to the Digital Investment Board by Registries to undertake the registry assignment, namely, that the requirement for private corporations to make an annual report filing gave Registries the means to guarantee a mandated pathway to compliance for a large percentage of the more than 550,000 BC corporations.
+  
+2. The annual report standalone filing is a better place to start for collecting beneficial ownership information. For example, it would require less development work initially than starting with incorporation.
+  
+3. The sooner we can stand up a working version of the Register workflow, the sooner we can get it in front of users in the dev environment for UX research purposes. This is crucial because the OCM component of Registering beneficial owners will be extensive and challenging.
+  
+4. The sooner we can get data into the beneficial owner database (even just sample data) the sooner we can extend SOLR to beneficial owner database and include it along with director search to create the integrated Person Search.
+
+# Detailed design
+
+## Include Add a Beneficial Owner workflow
+The AR standalone filing will contain the information in the current corporate annual report filing and will also include the workflow for adding beneficial owners. The workflow includes:
+1. Add Person section (name and email)
+2. Significant Individual Assessment section (evaluates user input and returns status)
+3. Add Personal Details (DOB, address, SIN, citizenship etc.)
```

Comment:
```suggestion
The workflow is only relevant for BCA entities.
1. Annual Report Date (this will be save to _lear_ or _colin_ as required)
2. Significant Individual Assessment section
3. UXA to determine:
   a. Add Beneficial Owner Details: OR
   b. Choose from Beneficial Ownership profile
4. Possibly create a Beneficial Ownership profile